### PR TITLE
feat: Transplant Near support

### PR DESCRIPTION
**Pull request by ukstv**
From https://github.com/ceramicnetwork/js-ceramic/pull/1728

Now tests do actually pass.

### CODE REVIEW

**Review by stbrody — COMMENTED** [review]
I'm not really very familiar with this code, but as best as I can tell it seems reasonable. My main concern is with the tests and making sure they don't depend on any external services.

File: packages/blockchain-utils-linking/src/__tests__/near.test.ts [hunk 1]
```diff
@@ -1,27 +1,22 @@
...
+const chainRef = 'testnet'
+const accountName = 'crustykitty.testnet'
+const keyPair = nearApiJs.utils.KeyPair.fromString(privateKey)
+const keyStore = new nearApiJs.keyStores.InMemoryKeyStore()
+keyStore.setKey(chainRef, accountName, keyPair)
+
+const config = {
+  keyStore, // instance of InMemoryKeyStore
+  networkId: 'testnet',
+  nodeUrl: 'https://rpc.testnet.near.org',
```
[the remark this answers is not in the document]
Comment (ukstv) [reply packages/blockchain-utils-linking/src/__tests__/near.test.ts:16]: That was my concern too. I ran the tests autonomously (wi-fi turned off), and they did pass indeed.

[the remark this answers is not in the document]
Comment (ukstv) [reply packages/blockchain-utils-linking/src/__tests__/near.test.ts:16]: So, I am not worried anymore about including this in the test code.

[the remark this answers is not in the document]
Comment (ukstv) [reply packages/blockchain-utils-linking/src/__tests__/near.test.ts:16]: @stbrody agree to push that to rc?

[the remark this answers is not in the document]
Comment (stbrody) [reply packages/blockchain-utils-linking/src/__tests__/near.test.ts:16]: > That was my concern too. I ran the tests autonomously (wi-fi turned off), and they did pass indeed.

Okay cool, that's good news.  If that's the case, then it seems that these URLs actually aren't being utilized at all.  Can we change them to something like "fakeurl.com" (or remove them entirely if possible?) so it's clear when reading the test file that these endpoints aren't actually being utilized?

[the remark this answers is not in the document]
Comment (ukstv) [reply packages/blockchain-utils-linking/src/__tests__/near.test.ts]: Left just one that is necessary for type checking. Others are removed.

**Review by stbrody — APPROVED** [review]
LGTM, thanks!

**Comment by Aarush-Goyal** [comment]
FInally NEAR compatibility with Ceramic

**Pull request merged**